mix.exs

```diff
@@ -53,7 +53,7 @@ defmodule Nerves.Network.Mixfile do
     [
       {:system_registry, "~> 0.7"},
       {:nerves_network_interface, "~> 0.4.4"},
-      {:nerves_wpa_supplicant, "~> 0.3"},
+      {:nerves_wpa_supplicant, ">= 0.4.0"},
```

Comment:
Allow anything up to but not including 1.0:
 
```suggestion
      {:nerves_wpa_supplicant, "~> 0.4"},
```

Reply:
This PR requires at least 0.4 but nothing lower than that. i thought ~> 0.4 would allow less than that because its a minor version?

Reply:
No, `~> 0.4` means at least 0.4.0 but not 1.0. If it were `~> 0.4.0`, then it would only allow between 0.4.0 and 0.4.X, but not 0.5.0. I've found that this causes people a lot of headaches when they're trying to bump other deps.

Reply:
ah okay perfect. I'll change this.